src/main/java/uk/gov/ons/census/action/model/entity/Case.java

```diff
@@ -92,4 +92,7 @@
 
   @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
   private boolean undeliveredAsAddressed;
+
+  @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
+  private boolean handDelivery;
```

Comment:
Should be `Boolean` and nullable if we're going to support non-CE1 messages having 'null' for hand delivery, instead of a misleading 'false' which isn't true (i.e. we don't ever hand deliver or 'not hand-deliver' a non-CE1 form)